devices/System.Buffers.Helpers/README.md

```diff
@@ -0,0 +1,6 @@
+# System.Buffers.Helpers
+
+Contains an `IBitConverter` interface with both Big and Little Endian implementations.
+Comes in utile when received data is encoded in another Endian way. Examples are:
+- Networking data which is encoded in Big Endian
+- serialized data could be encoded in another Endian way
```

Comment:
can you add a little example? the readme is published to the https://docs.nanoframework.net site and it's always better to get an example.

Reply:
Something like this?